services/route_manager/src/main/java/com/futurewei/alcor/route/service/Impl/RouterToDPMServiceImpl.java

```diff
@@ -50,7 +50,8 @@ public void sendInternalRouterInfoToDPM(InternalRouterInfo internalRouterInfo) t
         internalRouterInfos.add(internalRouterInfo);
         networkConfiguration.setInternalRouterInfos(internalRouterInfos);
         networkConfiguration.setRsType(Common.ResourceType.ROUTER);
-        networkConfiguration.setOpType(Common.OperationType.INFO);
+
+        networkConfiguration.setOpType(Common.OperationType.valueOf(internalRouterInfo.getOperationType().name()));
```

Comment:
name()?

Reply:
@DavidLiu506 I think you cannot use the value of `com.futurewei.alcor.common.enumClass.OperationType` directly maps to `com.futurewei.alcor.schema.Common.OperationType`, because their enum values are different.